Realm/RealmSwift/List.swift

```diff
@@ -19,17 +19,34 @@
 import Realm
 import Realm.Private
 
+/// Internal class. Do not use directly.
 public class ListBase: RLMListBase, Printable {
     // Printable requires a description property defined in Swift (and not obj-c),
     // and it has to be defined as @objc override, which can't be done in a
     // generic class.
+    /// Returns a human-readable description of the objects contained in the list.
     @objc public override var description: String { return _rlmArray.description }
+
+    /// Returns the number of objects in this list.
     public var count: Int { return Int(_rlmArray.count) }
 }
 
+/**
+`List<T>` is the container type in Realm used to define to-many relationships.
+
+Lists hold a single `Object` subclass (`T`) which defines the "type" of the list.
+
+Lists can be filtered and sorted with the same predicates as `Results<T>`.
+
+List properties on objects are lazily created when accessed, or can be
+obtained by querying a Realm.
```

Comment:
Also, List properties are created eagerly, since doing it lazily didn't seem to be possible.